File: 1.Python/Program17.py
# 5.Write a program which accept one number for user and check whether number is prime or not.
# Input :  5
# Output : It is Prime Number
def PrimeChk(No):
     iCnt=1
     if No>1:
         for iCnt in range(2, No):
            if (No % iCnt) == 0:
                return False
         else:
            return True
     else:
         return False
def Main():

   No=0
   bRet=0

   print("Enter the one number:")
   No =int(input())

   bRet=PrimeChk(No)
   if bRet==True:
       print("It is Prime Number")
   else:
       print("It is not Prime Number")

if __name__ == "__main__":
   Main()

File: 0.C_CPP_JAVA Programs/JAVA/Program17.java
/*
 2. Write a java  program which accept string from user and count number of small characters.
Input :   “Marvellous”
Output :   9

 */
import java.lang.*;
import java.util.*;
class DemoString
{
    int Count(String str)
    {
        char arr[] = str.toCharArray();
        int size=arr.length;
        int i=0;
        int iCount=0;
        while(i<size)
        {
            if(arr[i]>='a'&&arr[i]<='z')
            {
                iCount++;
            }
            i++;
        }
        return iCount;
    }
}
class Demo
{
    public static void main(String arg[])
    {

        Scanner sobj=new Scanner(System.in);
        System.out.println("Enter the string");
        String str=sobj.nextLine();

        DemoString dobj=new DemoString();

        int ret=dobj.Count(str);
        System.out.println("Small letter is: " + ret);
    }
}